src/notation/internal/notationundostack.cpp

```diff
@@ -136,17 +136,25 @@ void NotationUndoStack::commitChanges()
 
 void NotationUndoStack::lock()
 {
-    m_isLocked = true;
+    IF_ASSERT_FAILED(undoStack()) {
```

Comment:
@RomanPudashkin Now I get a crash here every time that I try to create a score by choosing instruments (not template). 
Problem seems to be that the `score()` of the (Master)Notation is not initialised yet.

Reply:
Sorry :( I'll check this problem